### PR TITLE
feat: rubymine, webstorm announcement

### PR DESCRIPTION
**Pull request by loujaybee**
- [ ] [Update header](https://github.com/gitpod-io/website/pull/2847#issuecomment-1269867454) based on @chrifro's feedback
- [x] [Update left-alignment](https://github.com/gitpod-io/website/pull/2847#issuecomment-1269867454) based on @chrifro's feedback
- [x] Emphasise the JetBrains "official" partnership
- [ ] Address @chrifro [comment about more text added to the preview](https://github.com/gitpod-io/website/pull/2847#issuecomment-1273066663)
- [x] Address comment from @mbrevoort [about "integrate don't dictate" copy update](https://github.com/gitpod-io/website/pull/2847/files#r993396403)

[**`Blog Preview`**](https://deploy-preview-2847--gitpod-kumquat.netlify.app/blog/webstorm-rubymine-now-in-beta)

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2847"></a>

### CODE REVIEW

**Comment by Siddhant-K-code** [comment]
I think we will need new `teaserImage`

File: src/routes/blog/webstorm-rubymine-now-in-beta.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+author: loujaybee
+date: Thursday, 04 Oct 2022 00:00:00 UTC
+excerpt: Lorem Ipsum
+image: jetbrains-beta.png
+slug: webstorm-rubymine-now-in-beta
+subtitle: Lorem
```
Comment (Siddhant-K-code) [inline src/routes/blog/webstorm-rubymine-now-in-beta.md:7]: Not Required i guess 🤔

File: src/routes/blog/webstorm-rubymine-now-in-beta.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+author: loujaybee
+date: Thursday, 04 Oct 2022 00:00:00 UTC
+excerpt: Lorem Ipsum
```
Comment (Siddhant-K-code) [inline src/routes/blog/webstorm-rubymine-now-in-beta.md:4]:
```suggestion
excerpt: Gitpod releases a beta version of RubyMine and WebStorm support.
```

**Comment by Siddhant-K-code** [comment]
This will be followed by #2852

**Review by mbrevoort — COMMENTED** [review]
Some initial comments in the top portion.

File: src/routes/blog/webstorm-rubymine-now-in-beta.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+author: loujaybee
+date: Thursday, 04 Oct 2022 00:00:00 UTC
+excerpt: As of today you can now use WebStorm and RubyMine to edit code in Gitpod!
+image: webstorm-rubymine-announcement-gitpod.png
+slug: webstorm-rubymine-now-in-beta
+subtitle: As of today you can now use WebStorm and RubyMine to edit code in Gitpod!
+teaserImage: webstorm-rubymine-announcement-gitpod.png
```
Comment (loujaybee) [inline src/routes/blog/webstorm-rubymine-now-in-beta.md:8]: image

CC: @chrifro - all good with this image? Any feedback or thoughts? 🙏

File: src/routes/blog/webstorm-rubymine-now-in-beta.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  export const prerender = true;
+</script>
+
+As of today you can now use WebStorm and RubyMine (in beta) to edit code in Gitpod.
+
+We've been working on the JetBrains IDE experience in Gitpod as part of our official partnership with JetBrains ([Gitpod x JetBrains join forces to solve 'works on my machine' problem](https://www.gitpod.io/blog/gitpod-jetbrains)).
+
+Here's a screencast to see exactly how working with JetBrains IDE and Gitpod looks.
+
+`youtube: 8djaRYT2FAY`
```
Comment (loujaybee) [inline src/routes/blog/webstorm-rubymine-now-in-beta.md:22]: This screencast is a bit old, I think we can still re-use though. 

CC: @akosyakov / @pawlean

File: src/routes/blog/webstorm-rubymine-now-in-beta.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+---
+author: loujaybee
+date: Thursday, 04 Oct 2022 00:00:00 UTC
+excerpt: As of today you can now use WebStorm and RubyMine to edit code in Gitpod!
+image: webstorm-rubymine-announcement-gitpod.png
+slug: webstorm-rubymine-now-in-beta
+subtitle: As of today you can now use WebStorm and RubyMine to edit code in Gitpod!
+teaserImage: webstorm-rubymine-announcement-gitpod.png
+title: RubyMine and Webstorm support now in Gitpod
```
Comment (loujaybee) [inline src/routes/blog/webstorm-rubymine-now-in-beta.md:9]:
```suggestion
title: RubyMine and WebStorm support now in Gitpod
```

**Comment by Siddhant-K-code** [comment]
On hold, because of https://github.com/gitpod-io/website/pull/2852

File: src/routes/blog/webstorm-rubymine-now-in-beta.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+`youtube: 8djaRYT2FAY`
+
+## Getting Started
+
+Here's how you can get started with a JetBrains IDE in Gitpod today:
+
+1. **Install [JetBrains Gateway](https://www.jetbrains.com/help/idea/remote-development-a.html#gateway)** - With the [JetBrains Gateway and Gitpod](/docs/integrations/jetbrains-gateway) plugin you can create and manage your latest 20 Gitpod workspaces.
+2. **Install the Gitpod plugin** - Open JetBrains Gateway and you'll see the Gitpod logo on the main page. Click "install" to install the [Gitpod plugin](https://plugins.jetbrains.com/plugin/18438-gitpod-gateway) for JetBrains Gateway.
+3. **Update your Gitpod preferences** - Select Rubymine or WebStorm on the [Gitpod preferences page](https://gitpod.io/preferences) which will set Rubymine or WebStorm as your default IDE for future workspace starts.
+4. **Start (or restart) your workspace** - Either start a workspace directly from within the [JetBrains Gateway](/docs/integrations/jetbrains-gateway) via the Gitpod plugin **OR** open a new workspace directly in Gitpod where on workspace start you will be prompted to open Rubymine or Webstorm for that workspace.
```
[the remark this answers is not in the document]
Comment (loujaybee) [reply src/routes/blog/webstorm-rubymine-now-in-beta.md]: > Keep it either of two:

Be careful reading too much into the intent. Both should be capitalised. 😄

**Comment by chrifro** [comment]
>@chrifro - all good with this image? Any feedback or thoughts? 🙏

the image looks great but it could be interpreted as RubyMine goes into Gitpod and Webstorm comes out of it :D 
Alternative (see [Figma](https://www.figma.com/file/gq1v6dj0rzs4tvJcFabKPy/Blog-%26-Changelog-images?node-id=1560%3A873))👇
Blog Post Workspace Networking - header

>Is it just me or does the center align and whitespace look kinda odd?

Nice catch! It's supposed to be left-aligned (see [Figma component library](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?node-id=690%3A1423)). Not sure what caused the change, but could you fix that? 🙇‍♀️

>Should I add more text? 😆 I'm not sure...

Would be great, if possible. A common habit is to simply use the first 1-2 sentences of the blog post

File: src/routes/blog/webstorm-rubymine-now-in-beta.md (hunk 4 above)
Comment (akosyakov) [reply src/routes/blog/webstorm-rubymine-now-in-beta.md:22]: fine with me

**Comment by loujaybee** [comment]
Thanks @chrifro ! 

> the image looks great but it could be interpreted as RubyMine goes into Gitpod and Webstorm comes out of it :D
Alternative (see [Figma](https://www.figma.com/file/gq1v6dj0rzs4tvJcFabKPy/Blog-%26-Changelog-images?node-id=1560%3A873))👇

@filiptronicek - Would you be able to pick up this amendment? 🙏 

> Not sure what caused the change, but could you fix that? 🙇‍♀️

@filiptronicek - Could you pick this up in a separate PR? Should be okay to raise and merge independently 🙏 

> Would be great, if possible. A common habit is to simply use the first 1-2 sentences of the blog post

👀 - Will look to update...

**Comment by Siddhant-K-code** [comment]
This will be merged after https://github.com/gitpod-io/website/pull/2852

**Comment by loujaybee** [comment]
@mbrevoort, @chrifro - are you able to check that you're happy that your comments are now addressed. 

If so can you approve the PR? Then we can ship 🙏

**Comment by mbrevoort** [comment]
I just pushed an edit to the branch updating the title and first paragraph.
image

File: src/routes/blog/webstorm-rubymine-now-in-beta.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+---
+author: loujaybee
```
Comment (loujaybee) [inline src/routes/blog/webstorm-rubymine-now-in-beta.md:2]: This needs updating to be @mustard-mh, @andreafalzetti, @felladrin, @loujaybee, (@akosyakov?)

Comment (akosyakov) [reply src/routes/blog/webstorm-rubymine-now-in-beta.md:2]: 👍

Comment (Siddhant-K-code) [reply src/routes/blog/webstorm-rubymine-now-in-beta.md:2]:
```suggestion
author: mustard-mh, andreafalzetti, felladrin, loujaybee, akosyakov
```

**Comment by Siddhant-K-code** [comment]
**performance-improvement (not related this blog change)**: In [`6ae8d22`](https://github.com/gitpod-io/website/pull/2847/commits/6ae8d22b0a5f3e23078acfdfafeeb6ce70ca7fcb), I 've introduced the `webp` file as our `teaserImage` & `image` instead of `png`.

image

**Pull request merged**